media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -2046,6 +2079,10 @@ VAStatus MediaLibvaCaps::CreateEncConfig(
                 return VA_STATUS_ERROR_UNSUPPORTED_RT_FORMAT;
             }
         }
+        if(VAConfigAttribContextPriority == attribList[j].type)
+        {
+            priority = attribList[j].value - CONTEXT_PRIORITY_MAX/2;
```

Comment:
same

Reply:
Done the PR: https://github.com/intel/media-driver/pull/1020